src/core/ext/filters/http/client/http_client_filter.cc

```diff
@@ -67,6 +67,8 @@ struct call_data {
   grpc_closure on_send_message_next_done;
   grpc_closure* original_send_message_on_complete;
   grpc_closure send_message_on_complete;
+  grpc_error* recv_trailing_metadata_err;
```

Comment:
s/err/error/

(Naming rules at https://google.github.io/styleguide/cppguide.html#General_Naming_Rules say to avoid abbreviations.)